src/main/java/tds/exam/results/services/impl/ExamResultsServiceImpl.java

```diff
@@ -65,20 +69,26 @@ public TDSReport findAndSendExamResults(final UUID examId) {
                 findAndSendExamResults(examId, report);
             } catch (Exception e) {
                 // Log error, and do not rethrow to prevent ERT from re-processing this same request
-                log.error("Error occurred while processing or sending the exam results for examId {}: " + e.getStackTrace());
+                log.error("Error occurred while processing or sending the exam results for examId {}: {}", examId,
+                    e.getMessage());
```

Comment:
Please pass along the Exception in addition to the message.  That way we get the full stack trace when it is logged.  Student follows the pattern you have above and it has made it hard to track down errors just from looking at the logs.